frontend/src/app/output/page-content/open-api-end-points/open-api-end-points.component.ts

```diff
@@ -13,9 +14,14 @@ export class OpenApiEndPointsComponent implements OnInit,AfterViewInit {
   @Input() openApiPathJson: OpenApiPath;
   @Input() position: number;
 
-   id:string;
+  id:string;
+  proxyUrl:string;
 
-  constructor() { }
+  constructor(private applicationService: ConfigService) {
+    this.applicationService.getConfigs().subscribe(result => {
```

Comment:
I would do it on the `ngOnInit`. Waiting @loriepisicchio thought on this but I believe constructor is not the place to call other services.